js/bulkhead-callback.js

```diff
@@ -279,16 +283,73 @@ var bulkheadCallBack = (function() {
         return match;
     };
 
+    var __validateEditorContent_BulkheadStep = function(content) {
+        var match = false;
+        try {
+            var pattern = "@Bulkhead\\(50\\)\\s*public Service serviceForVFA";
+            var regExpToMatch = new RegExp(pattern, "g");
+            content.match(regExpToMatch)[0];
+            match = true;
+        } catch (ex) {
+
+        }
+        return match;
+    };
+
+    var __validateEditorContent_AsyncBulkheadStep = function(content) {
+        var match = false;
+        try {
+            var pattern1 = "@Asynchronous\\s*@Bulkhead\\(value\\s?=\\s?50,\\s*" + 
+                "waitingTaskQueue\\s?=\\s?50\\)\\s*" +
+                "public Future<Service> serviceForVFA";
+            var regExp1 = new RegExp(pattern1, "g");
+
+            var pattern2 = "Service chatService = new ChatSession\\(counterForVFA\\);\\s*" + 
+                "return CompletableFuture\\.completedFuture\\(chatService\\);";
+            var regExp2 = new RegExp(pattern2, "g");
+
+            var pattern3 = "counterForVFA\\s*=\\s*0;\\s*" +
+                "public Future<Service> requestForVFA\\(\\)\\s*{\\s*" +
+                "counterForVFA\\+\\+;\\s*" +
+                "return serviceForVFA\\(counterForVFA\\);";
+            var regExp3 = new RegExp(pattern3, "g");
+
+            content.match(regExp1)[0];
+            content.match(regExp2)[0];
+            content.match(regExp3)[0];
+            match = true;
+        } catch (ex) {
+
+        }
+        return match;
+    };
+
+    var __validateEditorContent_FallbackStep = function(content) {
+        var match = false;
+        try { 
+            var pattern = "return serviceForVFA\\(counterForVFA\\);\\s*" +
+            "}\\s" + 
+            "(?:\\s*.*){0,5}" + //0-5 instances of repeating group of any chars and whitespace including new lines
```

Comment:
Sorry I am not following the logic on line 332. Why 0 to 5 instances and why any chars?

Reply:
So this is probably not needed for this step because we only allow the user to type on a single line, but it would be useful in the playground. 

The @Asynchronous, @Bulkhead, and @Fallback annotations should be possible in any order. While this isn't a foolproof check, it allows a little leeway for where they add the @Fallback annotation.

Since it is not necessary in this step, I can remove it.